edx_name_affirmation/models.py

```diff
@@ -11,6 +12,32 @@
 User = get_user_model()
 
 
+class VerifiedNameStatus(str, Enum):
+    """
+    Possible states for the verified name.
+
+    Pending: the verified name has been created, but not seen by anyone
+
+    Submitted: the verified name has been submitted to a verification authority
+
+    Approved, Denied: resulting states from that authority
+
+    State transitions in the verifying processes may not move this status.
+    For example in proctoring the verification might be pending at exam start,
+    unchanged when the exam is ready to submit, move to submitted on submit,
+    and then remain unchanged through multiple verification statuses until
```

Comment:
That is exactly what I meant but was not clear enough.

Maybe instead of this something along the lines of "this is the status for the verified name, which is separate from the status of verification processes such as proctoring"?

Reply:
4295b59